README.md

```diff
@@ -1 +1,34 @@
-# Hello-Microverse
\ No newline at end of file
+# Hello-Microverse-Project
+This project is meant for practicing basic HTML and CSS.
+
+Built With   
+HTML
+CSS
+
+## Getting Started
+Usage
+Open the index.html file in the repository to see the output.
```

Comment:
- [ ] [OPTIONAL] It would be nice adding more content to this section like how to clone the project, tools required to run it locally and how to run it.